src/System.Net.Requests/tests/project.json

```diff
@@ -4,8 +4,11 @@
     "System.IO.Compression": "4.1.0-rc3-24113-00",
     "System.Linq.Expressions": "4.1.0-rc3-24113-00",
     "System.Net.Http": "4.1.0-rc3-24113-00",
+    "System.Net.NetworkInformation": "4.1.0-rc3-24113-00",
     "System.Net.Primitives": "4.0.11-rc3-24113-00",
     "System.Net.Requests": "4.0.11-rc3-24113-00",
+    "System.Net.Security": "4.0.0-rc3-24113-00",
+    "System.Net.Sockets": "4.1.0-rc3-24113-00",
```

Comment:
By the time you see this, I will have hopefully merged the package update to fix the SslStream issues.  At that point you'll need to manually bump these new entries up from 113 to 116.